invenio_search_ui/ext.py

```diff
@@ -49,6 +49,11 @@ def init_app(self, app):
     def init_config(self, app):
         """Initialize configuration."""
         app.config.setdefault(
-            "SEARCH_UI_BASE_TEMPLATE",
-            app.config.get("BASE_TEMPLATE",
-                           "invenio_search_ui/base.html"))
+            'SEARCH_UI_BASE_TEMPLATE', 'invenio_theme/page.html'
+        )
+        app.config.setdefault(
+            'SEARCH_UI_SEARCH_API', None
```

Comment:
IMHO this should not be necessary and it might cause issue if you want to check that it was set before.